tests/NamespaceFullyQualifiedTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Opis\Closure\Test;
+
+use Closure;
+use Opis\Closure\ReflectionClosure;
+
+final class NamespaceGroupTest extends \PHPUnit\Framework\TestCase
+{
+    public function test_namespace_fully_qualified()
```

Comment:
More tests needed here

Reply:
This test was just a duplicate of tests already written in other files. The reason I copied it here was so that I would have less Tokens when debugging what was crashing and adding more cases to the parser.